lib/Chunk.js

```diff
@@ -425,22 +425,18 @@ class Chunk {
 
 	/**
 	 * @param {ChunkGroup} chunkGroup the chunkGroup the chunk is being added
-	 * @returns {boolean} returns true if chunk is not a part of chunkGroup and is added successfully
+	 * @returns {void}
 	 */
 	addGroup(chunkGroup) {
-		if (this._groups.has(chunkGroup)) return false;
 		this._groups.add(chunkGroup);
-		return true;
```

Comment:
seems like a breaking change for webpack 4
e.g. some plugins based on this result

Reply:
yes